StPicoEvent/StPicoEvent.h

```diff
@@ -110,6 +111,8 @@ class StPicoEvent : public TObject
   UShort_t mFillId;
   Float_t  mBField;
 
+  Int_t    mTime;//To set timestamp for St_db_Maker. This is what StMuDstMaker used to GetEvtHddr()->SetGMTime(cast into unsigned int). Peifeng Liu
```

Comment:
@pliuphys 
Do we ever use event timestamp? If we are talking about calibration and other corrections, these are usually done on per run basis. St_db_Maker knows extracts correct data from the DB based on the run number, we don't need to provide any additional timestamps in this workflow